packages/esm-patient-common-lib/src/workspaces.ts

```diff
@@ -1,20 +1,17 @@
 import { useCallback } from 'react';
 import { type DefaultWorkspaceProps, launchWorkspace, navigateAndLaunchWorkspace } from '@openmrs/esm-framework';
-import { getPatientUuidFromStore, usePatientChartStore } from './store/patient-chart-store';
+import { usePatientChartStore } from './store/patient-chart-store';
 import { launchStartVisitPrompt } from './launchStartVisitPrompt';
 import { useSystemVisitSetting } from './useSystemVisitSetting';
 import { useVisitOrOfflineVisit } from './offline/visit';
 
 export interface DefaultPatientWorkspaceProps extends DefaultWorkspaceProps {
+  patient: fhir.Patient;
   patientUuid: string;
 }
 
 export function launchPatientWorkspace(workspaceName: string, additionalProps?: object) {
```

Comment:
Do we still need this function? It looks like we can safely replace it with a direct call to `launchWorkspace`.